### ISSUE TITLE
Remove top-level await workaround

### ISSUE THREAD

**Issue by sindresorhus** [SamVerschueren/tsd#35]
Looks like TS 3.7 will get built-in support, so we can remove this workaround at some point.

https://httptoolkit.tech/blog/5-big-features-of-typescript-3.7/

**Comment by SamVerschueren** [comment]
Jup, read it too :). Thanks for opening this issue so we don't forget.

**Comment by SamVerschueren** [comment]
I believe we can ditch this now but only if we set the `module` to `esnext` if I'm not mistaken.

**Comment by sindresorhus** [comment]
Right, so we should probably wait until we can use ESM natively in Node.
